pinot-controller/src/main/java/org/apache/pinot/controller/helix/core/assignment/segment/OfflineSegmentAssignment.java

```diff
@@ -88,7 +89,12 @@ public void init(HelixManager helixManager, TableConfig tableConfig) {
     if (_partitionColumn == null) {
       LOGGER.info("Initialized OfflineSegmentAssignment with replication: {} without partition column for table: {} ",
           _replication, _offlineTableName);
+      _useDeterministicSegmentAssignment = false;
     } else {
+      // TODO: Where do I keep the segment assignment strategy
```

Comment:
Where's the right place to keep the segment assignment strategy constant?

I saw CommonConstants has a SegmentAssignmentStrategyType enum, but it's not getting used anywhere.